### ISSUE TITLE
Frame data missing or invalid

### ISSUE THREAD

**Issue by LazyLemons** [python-hyper/hyper-h2#357]
`
kcelik@kcelik-pc:~/http2_server$ hyper GET http://http2bin.org/get
Traceback (most recent call last):
  File "/usr/local/bin/hyper", line 9, in <module>
    load_entry_point('hyper==0.7.0', 'console_scripts', 'hyper')()
  File "/usr/local/lib/python2.7/dist-packages/hyper/cli.py", line 259, in main
    data = request(args)
  File "/usr/local/lib/python2.7/dist-packages/hyper/cli.py", line 249, in request
    response = conn.get_response()
  File "/usr/local/lib/python2.7/dist-packages/hyper/common/connection.py", line 140, in get_response
    self._conn._connect_upgrade(e.sock)
  File "/usr/local/lib/python2.7/dist-packages/hyper/http20/connection.py", line 400, in _connect_upgrade
    self._recv_cb()
  File "/usr/local/lib/python2.7/dist-packages/hyper/http20/connection.py", line 771, in _recv_cb
    self._single_read()
  File "/usr/local/lib/python2.7/dist-packages/hyper/http20/connection.py", line 669, in _single_read
    events = conn.receive_data(data)
  File "/usr/local/lib/python2.7/dist-packages/h2/connection.py", line 1298, in receive_data
    for frame in self.incoming_buffer:
  File "/usr/local/lib/python2.7/dist-packages/h2/frame_buffer.py", line 165, in next
    raise FrameDataMissingError("Frame data missing or invalid")
h2.exceptions.FrameDataMissingError: Frame data missing or invalid
`

Python 2.7

**Comment by manjitborah** [comment]
Hi, Did you find a solution to this issue? 
I am getting the same error using hyper 0.7.0

**Comment by merutak** [comment]
I had this problem: apparently there's a bug (?) in Python 2.7.3, fixed in 2.7.6 (or somewhere in between), that `struct.unpack` doesn't work with `memoryview`. So (if this is the same problem that I had) one solution is to upgrade to Python 2.7.6. Another solution is to change the source of hyper:

```
h2/frame_buffer.py
-163,1 +163,2
-                f.parse_body(memoryview(self.data[9:9+length]))
+                # untested, use at your own risk
+                f.parse_body(self.data[9:9+length])
```
